Orange/widgets/visualize/owheatmap.py

```diff
@@ -462,6 +463,7 @@ class Information(widget.OWWidget.Information):
         discrete_ignored = Msg("{} discrete column{} ignored")
         row_clust = Msg("{}")
         col_clust = Msg("{}")
+        sparse_densified = Msg("Data was converted to dense format")
```

Comment:
I've shorten your version to simply: Showing this data may require a lot of memory.